### ISSUE TITLE
Improve the "Review your annotations" summary view

### ISSUE THREAD

**Issue by boazsender** [bocoup/image-annotator#165]
This is a little tough to parse. @isaacdurazo do you have any suggestions for how to improve this?:

![ajl-summary](https://cloud.githubusercontent.com/assets/122117/24676187/039e26c2-1950-11e7-86b3-39064af94b55.png)

**Comment by isaacdurazo** [comment]
I agree @boazsender Let me think a little bit about what we can do to improve it
